apps/platform/.eslintrc.js

```diff
@@ -1,7 +1,11 @@
 /** @type {import("eslint").Linter.Config} */
 module.exports = {
   root: true,
-  extends: ["@muqa/eslint-config/next.js"],
+  extends: [
+    "@muqa/eslint-config/next.js", 
```

Comment:
This one is already in `@muqa/eslint-config/next.js`.

Reply:
removed, thx